pennylane/math/quantum.py

```diff
@@ -938,3 +938,47 @@ def _check_state_vector(state_vector):
     if not is_abstract(norm):
         if not allclose(norm, 1.0, atol=1e-10):
             raise ValueError("Sum of amplitudes-squared does not equal one.")
+
+
+def min_entropy(state, base=None, check_state=False, c_dtype="complex128"):
+    r"""
+    Compute the min-entropy of a state.

```

Comment:
It would be good to include a brief explanation or equation describing what min-entropy is, in case the user is not familiar with it.

Reply:
Sure I will add them too. Thanks for suggestions 😸